test/ngRoute/directive/ngViewSpec.js

```diff
@@ -120,6 +120,26 @@ describe('ngView', function() {
   });
 
 
+  it('should use default resolve controller when resolve map defined with no controller', function() {
```

Comment:
The test description is kibd of undescriptive (I wouldn't have guessed what's supposed to do just by reading the description).